### PR TITLE
Minimal Mesos console template

### PR DESCRIPTION
**Pull request by StephanErb**
Implemented functionality: 
- Plotting of CPU and RAM usage per slave.
- Stats for requested vs actual used resources

This pull request is partially addressing #9. Even though this is a very rudimentary console template, I believe we can improve it over time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,4 +45,10 @@ It's also possible to use in your own Dockerfile:
 
     FROM prom/mesos-exporter
 
+
+## Console Dashboard
+
+Copy the content of `consoles` to the consoles folder used by your Prometheus master. Your Mesos
```
Comment (brian-brazil) [inline README.md:51]: These currently live in the prometheus repo, as there's more changes than this required for it to hook into the menu and table.

File: consoles/mesos.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{{ template "head" . }}
+
+{{ template "prom_right_table_head" }}
+<tr>
+    <th colspan="2">Statistics</th>
+</tr>
+<tr>
+  <td>Mesos Jobs (active)</td>
```
Comment (brian-brazil) [inline consoles/mesos.html:8]: In line with the other consoles, this should start with up/total mesos exporters

File: consoles/mesos.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  <td>{{ template "prom_query_drilldown" (args "count(mesos_task_cpu_limit{job='mesos'})") }}</td>
+</tr>
+<tr>
+  <td>Memory (used / allocated)</td>
```
Comment (brian-brazil) [inline consoles/mesos.html:12]: I'd do separate rows for these

File: consoles/mesos.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    yUnits: 'B'
+  })
+  </script>
+
+  <h3>CPU Usage</h3>
+  <div id="cpu"></div>
+  <script>
+  new PromConsole.Graph({
+    node: document.querySelector("#cpu"),
+    expr: "sum(rate(mesos_task_cpu_user_seconds_total[2m])) by (slave) + sum(rate(mesos_task_cpu_system_seconds_total[2m])) by (slave)",
```
Comment (brian-brazil) [inline consoles/mesos.html:49]: We use 5m in consoles currently, we can't make assumptions about the users' scrape interval

File: consoles/mesos.html (hunk 3 above)
Comment (brian-brazil) [inline consoles/mesos.html:12]: I'm not sure this is meaningful in Mesos terms, as used is allocated. Allocated compared to the value in --resources would be the more appropriate metric, as that's how much of the slave's resources is reserved.

**Comment by StephanErb** [comment]
Thanks for the feedback. Unfortunately I am totally under water right now. Will reconsider when #11 and #13 have been merged.

**Pull request closed without merging**